easybuild/tools/options.py

```diff
@@ -239,8 +239,10 @@ def config_options(self):
             # purposely take a copy for the default logfile format
             'logfile-format': ("Directory name and format of the log file",
                                'strtuple', 'store', DEFAULT_LOGFILE_FORMAT[:], {'metavar': 'DIR,FORMAT'}),
+            'module-family': ("Include family statement in generated module file", 'str', 'store', None),
```

Comment:
no checks on what is supported?
can you give an example what could/should be added here?